addons/point_of_sale/static/src/app/screens/ticket_screen/ticket_screen.js

```diff
@@ -242,21 +243,14 @@ export class TicketScreen extends Component {
 
         const partner = order.get_partner();
 
-        const allToRefundDetails = this._getRefundableDetails(partner);
-        if (allToRefundDetails.length == 0) {
+        const allToRefundDetails = this._getRefundableDetailsGroupedByOrder(partner)[order.uid];
+
+        if (!allToRefundDetails) {
             this._state.ui.highlightHeaderNote = !this._state.ui.highlightHeaderNote;
             return;
         }
 
-        // The order that will contain the refund orderlines.
-        // Use the destinationOrder from props if the order to refund has the same
-        // partner as the destinationOrder.
-        const destinationOrder =
-            this.props.destinationOrder &&
-            partner === this.props.destinationOrder.get_partner() &&
-            !this.pos.doNotAllowRefundAndSales()
-                ? this.props.destinationOrder
```

Comment:
If you change this line, I guess we could remove the props `destinationOrder` as it is not used anywhere else I think

Reply:
i ended up not changing the line, so i'll keep the `props`